proposals/nnnn-remote-control-seat.md

```diff
@@ -0,0 +1,233 @@
+  <param name="coolingLevel" type="Integer" minvalue="0" maxvalue="100" mandatory="false"></param>
+
+  <param name="horizontalPosition" type="Integer" minvalue="0" maxvalue="100" mandatory="false"></param>
+  <param name="verticalPosition" type="Integer" minvalue="0" maxvalue="100" mandatory="false"></param>
+  <param name="frontVerticalPosition" type="Integer" minvalue="0" maxvalue="100" mandatory="false"></param>
+  <param name="backVerticalPosition" type="Integer" minvalue="0" maxvalue="100" mandatory="false"></param>
+  <param name="backTiltAngle" type="Integer" minvalue="0" maxvalue="100" mandatory="false"></param>
+  
+  <param name="headSupportHorizontalPosition" type="Integer" minvalue="0" maxvalue="100" mandatory="false"></param>
+  <param name="headSupportVerticalPosition" type="Integer" minvalue="0" maxvalue="100" mandatory="false"></param>
+  
+  <param name="massageEnabled" type="Boolean" mandatory="false"></param>
+  <param name="massageMode" type="MassageModeData" minsize="1" maxsize="2" array="true" mandatory="false"></param>
+  <param name="massageCushionFirmness" type="MassageCushionFirmness" minsize="1" maxsize="5" array="true" mandatory="false"></param>
+  
+  <param name="memory" type="SeatMemoryAction" mandatory="false"></param>
+</struct>
+
+<struct name="SeatControlCapabilities">
+  <param name="moduleName" type="String" maxlength="100"  mandatory="true">
+    <description>
+    The short friendly name of the light control module.
+    It should not be used to identify a module by mobile application.
+    </description>
+  </param>
+  <param name="heatingEnabledAvailable" type="Boolean" mandatory="false">
+  </param>
+  <param name="coolingEnabledAvailable" type="Boolean" mandatory="false">
+  </param>
+  <param name="heatingLevelAvailable" type="Boolean" mandatory="false">
+  </param>
+  <param name="coolingLevelAvailable" type="Boolean" mandatory="false">
+  </param>
+  <param name="horizontalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="verticalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="frontVerticalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="backVerticalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="backTiltAngleAvailable" type="Boolean" mandatory="false">
+  </param>
+  <param name="headSupportHorizontalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="headSupportVerticalPositionAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="massageEnabledAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="massageModeAvailable" type="Boolean" mandatory="false">
+  </param> 
+  <param name="massageCushionFirmnessAvailable" type="Boolean" array="false">
+  </param>
+  <param name="memoryAvailable" type="Boolean" array="false">
```

Comment:
Should 'array="false"' be 'mandatory="false"'?

Reply:
changed, thank you